package.json

```diff
@@ -41,7 +41,8 @@
   "peerDependencies": {
     "@apollo/client": "^3.3.19",
     "graphql": "^15.5.0",
-    "react": "^16.8.0 || ^17.0.0"
+    "react": ">= 16.8.0",
+    "react-dom": ">= 17.0.2"
```

Comment:
I don't think react-dom is needed in the peer dependency - SDK is not using it directly